Upstream fixes already included in the latest community updates to coolkey v1.1.0

Fixes a syntax error when running configure script.

--- a/configure.in	(revision 53)
+++ b/configure.in	(revision 81)
@@ -124,9 +124,9 @@
 if test $WINDOWS -ne 1; then
   PKG_CHECK_MODULES(NSS, nss, true, [ AC_MSG_ERROR(could not find NSS Crypto libraries) ])
 fi
-  enable_pk11install = "yes"
+  enable_pk11install="yes"
 else
-  enable_pk11install = "no"
+  enable_pk11install="no"
   AC_MSG_WARN([skipping pk11install])
 fi
 
